test/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -38,6 +38,12 @@ class TestCommon {
                 "ItMustNotExceed80BytesInLength"
         const val MAX_RELAYSTATE_LEN = 80
 
+        const val REQUESTER = "urn:oasis:names:tc:SAML:2.0:status:Requester"
+        const val VERSION_MISMATCH = "urn:oasis:names:tc:SAML:2.0:status:VersionMismatch"
+        private const val SUCCESS = "urn:oasis:names:tc:SAML:2.0:status:Success"
```

Comment:
❓ Do you intend to use the `SUCCESS` and `RESPONDER` consts at a later date?

Reply:
We will need `RESPONDER` for the SP tests and not sure right now about `SUCCESS`